portal/common/lib/page_fetching.bench.ts

```diff
@@ -6,13 +6,20 @@ import { fetchPage } from './page_fetching';
 import { SuiClient, getFullnodeUrl } from '@mysten/sui/client'
 import { NETWORK } from './constants';
 
+const LANDING_PAGE_OID = '0x5fa99da7c4af9e2e2d0fb4503b058b9181693e463998c87c40be78fa2a1ca271';
+const FLATLAND_OID = '0x049b6d3f34789904efcc20254400b7dca5548ee35cd7b5b145a211f85b2532fa';
```

Comment:
Suggestion: this is the object ID of the flatland mint page, which is an interesting thing to bench. However, we had discussed to bench also one of the flatlander sites, so that we bench redirects as well. Can you add one of those here? (mint a new one and copy the object id?)

Reply:
You're right, I missed this one! I will add it asap 👍